src/main/java/edu/cornell/kfs/module/ezra/service/impl/EzraServiceImpl.java

```diff
@@ -365,8 +365,8 @@ public Agency createAgency(Long sponsorId) {
 		Agency agency = new Agency();
 		agency.setAgencyNumber(sponsorId.toString());
 		agency.setReportingName(sponsor.getSponsorLabel());
-		if (ObjectUtils.isNotNull(sponsor.getSponsorName()) && sponsor.getSponsorName().length() > 50)
-			agency.setFullName(sponsor.getSponsorName().substring(0,49));
```

Comment:
Should the second substring() argument be 150 instead of 149? I know the old code had 49 instead of 50, but the ending index for substring() calls is an exclusive index, not an inclusive one.